Classes/BITCrashCXXExceptionHandler.mm

```diff
@@ -58,7 +58,7 @@
 
 @implementation BITCrashUncaughtCXXExceptionHandlerManager
 
-extern "C" void LIBCXXABI_NORETURN __cxa_throw(void *exception_object, std::type_info *tinfo, void (*dest)(void *))
```

Comment:
Could replace with `__attribute__((noreturn))`
FYI, this seems to be when it was changed: http://llvm.org/viewvc/llvm-project?view=revision&revision=296567